### PR TITLE
Some new headlines

### PR DESCRIPTION
**Pull request by mikejbrown**
Just a bit of flavour text. I know some are a little long. They seem to be up for a reasonable time though and I figure that's part of the fun. :grinning:

### CODE REVIEW

**Comment by kd7uiy** [comment]
Works for me. I'm not so sure about the Lorem Ipsum, but...

File: Assets/StreamingAssets/Data/Headlines.xml [hunk 1]
```diff
@@ -4,4 +4,17 @@
...
+    <Headline>As the inevitability of his impending death began to crystalise in his mind, Rupert unnacountably wondered whether it would be classified as "W40.9XXD Explosion of unspecified explosive materials (subsequent encounter)" or, he hoped, the more colourful "W59.22XA Struck by turtle (initial encounter)" under the revised and expanded ICD-10-CM American medical billing code scheme.</Headline>
+    <Headline>Attention all Quillcorp customers: Quillcorp security wishes to inform you that everything is okay. There is nothing wrong. Not even a little bit. Please be reassured that staff are doing all that they can to ensure that nothing remains to be amiss, continuing the situation of complete normality and lack of cause for alarm as was previously mentioned.</Headline>
+    <Headline>Quillcorp invites all customers who feel even the slightest bit of unease to please remain calm. Claims that a neurotoxin has been inadvertently released into the air supply are likely to be grossly exaggerated.</Headline>
+    <Headline>Lorem Ipsum</Headline>
```
Comment (kd7uiy) [inline Assets/StreamingAssets/Data/Headlines.xml]: I think this particular news isn't a great example. If you are going to do it, do it right and include at least a few sentences of Lorem Ipsum, but...

Comment (mikejbrown) [inline Assets/StreamingAssets/Data/Headlines.xml]: The idea is that the Quillcorp news editors are, well.... out to lunch a lot. I'll extend the lorem ipsum, see how that works?

File: Assets/StreamingAssets/Data/Headlines.xml [hunk 2]
```diff
@@ -4,4 +4,17 @@
     <Headline>Notice: Quillcorp has placed an embargo on "Chairs", any Quillcorp Basic Utility Station found in possession of this illegal contraband will be fined 2 million Quillbucks.</Headline>
     <Headline>Introducing the MagBlast 2200 magnetic rifle. 70% of the time, it hits every time.</Headline>
     <Headline>Thinktank finds something that could go wrong. Head of QuillCorp not available for comment.</Headline>
+    <Headline>Attention: Quillcorp has banned all video games except for ProjectPorcupine. Any person caught playing a banned game will face severe penalties.</Headline>
+    <Headline>As the inevitability of his impending death began to crystalise in his mind, Rupert unnacountably wondered whether it would be classified as "W40.9XXD Explosion of unspecified explosive materials (subsequent encounter)" or, he hoped, the more colourful "W59.22XA Struck by turtle (initial encounter)" under the revised and expanded ICD-10-CM American medical billing code scheme.</Headline>
```
Comment (TomMalbran) [inline Assets/StreamingAssets/Data/Headlines.xml]: crystalise -> crystallise

File: Assets/StreamingAssets/Data/Headlines.xml [hunk 3]
```diff
@@ -4,4 +4,17 @@
...
+    <Headline>Quillcorp invites all customers who feel even the slightest bit of unease to please remain calm. Claims that a neurotoxin has been inadvertently released into the air supply are likely to be grossly exaggerated.</Headline>
+    <Headline>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</Headline>
+    <Headline>[redacted] has denied claims that [redacted]. All personnel are cautioned that [redacted].</Headline>
+    <Headline>Quillcorp reminds all staff that the Quillcorp brand cloning pod is strictly illegal in most municipalities. Do not mention the cloning pod.</Headline>
+    <Headline>This just in: a SPACE dog has been discovered!</Headline>
+    <Headline>This just in: man bites SPACE dog!</Headline>
+    <Headline>A Kessler swarm of typewriters has been spotted incoming. Do not worry, there is only a small risk of writing Hamlet by chance.</Headline>
+    <Headline>Quillcorp is asking for volunteers to take part in a vacuum breathing study.</Headline>
+    <Headline>Quilcrop news edtors go on stirk.</Headline>
```
Comment (TomMalbran) [inline Assets/StreamingAssets/Data/Headlines.xml]: edtors -> editors
stirk -> strike

Comment (kd7uiy) [inline Assets/StreamingAssets/Data/Headlines.xml]: I noticed the same thing, but I decided eventually it is deliberate.

Comment (mikejbrown) [inline Assets/StreamingAssets/Data/Headlines.xml]: that's the point :P

Comment (TomMalbran) [inline Assets/StreamingAssets/Data/Headlines.xml]: Oh. I get it now. :P

Comment (mikejbrown) [inline Assets/StreamingAssets/Data/Headlines.xml]: :laughing:

File: Assets/StreamingAssets/Data/Headlines.xml [hunk 4]
```diff
@@ -4,4 +4,18 @@
     <Headline>Notice: Quillcorp has placed an embargo on "Chairs", any Quillcorp Basic Utility Station found in possession of this illegal contraband will be fined 2 million Quillbucks.</Headline>
     <Headline>Introducing the MagBlast 2200 magnetic rifle. 70% of the time, it hits every time.</Headline>
     <Headline>Thinktank finds something that could go wrong. Head of QuillCorp not available for comment.</Headline>
+    <Headline>Attention: Quillcorp has banned all video games except for ProjectPorcupine. Any person caught playing a banned game will face severe penalties.</Headline>
+    <Headline>As the inevitability of his impending death began to crystallize in his mind, Rupert unaccountably wondered whether it would be classified as "W40.9XXD Explosion of unspecified explosive materials (subsequent encounter)" or, he hoped, the more colourful "W59.22XA Struck by turtle (initial encounter)" under the revised and expanded ICD-10-CM American medical billing code scheme.</Headline>
```
Comment (bjubes) [inline Assets/StreamingAssets/Data/Headlines.xml]: "As the inevitability.." isn't really a headline, I think its a bit too long and doesn't fit the rest of the additions

Comment (kd7uiy) [inline Assets/StreamingAssets/Data/Headlines.xml]: I agree, it's more of a story...

File: Assets/StreamingAssets/Data/Headlines.xml [hunk 5]
```diff
@@ -4,4 +4,17 @@
     <Headline>Notice: Quillcorp has placed an embargo on "Chairs", any Quillcorp Basic Utility Station found in possession of this illegal contraband will be fined 2 million Quillbucks.</Headline>
     <Headline>Introducing the MagBlast 2200 magnetic rifle. 70% of the time, it hits every time.</Headline>
```
Comment (ComputerOverlord) [inline Assets/StreamingAssets/Data/Headlines.xml]: Hahahah

**Pull request merged**